homework/myworld/web_scrapper.py

```diff
@@ -0,0 +1,91 @@
+import psycopg2
+import requests
+import re
+from bs4 import BeautifulSoup, element
+import os
+        
+# For the credentials mentioned below, you may refer the docker-compose.yml present in myworld .
+db_name = 'employee_db'
+db_user = 'postgres'
+db_pass = '123456'
+db_host = 'psql-db2'
+db_port = '5432'
+        
+# This will create the connection the to postgres database.
+conn = psycopg2.connect(dbname=db_name, user=db_user, password=db_pass, host=db_host, port=db_port)
+        
+        
+def add_row_to_blog(title, author, date, time,content,recommended,path):
+    # This function will add the entry to database
+    sql = """INSERT INTO members_blog (title, release_date, blog_time, author, created_date, content, recommended, path) VALUES (%s, %s::DATE, %s::TIME, %s, NOW(),%s,%s,%s)"""
+            
+    with conn:
```

Comment:
@rajashree-k Why `with` is used? what is `with`?

Reply:
with statement is used for exception handling. So it make the code cleaner and much more readable.

Reply:
@rajashree-k could you please explain in detail, what exception handling you're referring to?
( I don't mind even if you write 100 lines of explanation but if you're not sure so as to why `with` is used, please look up on the internet).

Reply:
with statement is closes the connection and cursor automatically.